test-support/reference-trie/Cargo.toml

```diff
@@ -13,7 +13,7 @@ hash256-std-hasher = { path = "../../hash256-std-hasher", version = "0.15.2" }
 keccak-hasher = { path = "../keccak-hasher", version = "0.15.3" }
 trie-db = { path = "../../trie-db", default-features = false, version = "0.22.0" }
 trie-root = { path = "../../trie-root", default-features = false, version = "0.16.0" }
-parity-scale-codec = { version = "2.2.0-rc.2", features = ["derive"] }
+parity-scale-codec = { version = "2", features = ["derive"] }
```

Comment:
I'm not sure about this cc @Xanewok

Reply:
Let's land this, we'll probably pursue new minor version + max-encoded-len via feature gate as outlined in https://github.com/paritytech/parity-common/pull/552#issuecomment-872252537

Reply:
This will probably require a version bump now?

Reply:
Alternatively, we can yank `0.23.1` and trie-bench `0.27.1`